truss/remote/truss_remote.py

```diff
@@ -0,0 +1,64 @@
+from abc import ABC, abstractmethod
+from typing import Dict
+
+import requests
+from truss.truss_handle import TrussHandle
+
+
+class TrussService(ABC):
+    """
+    TODO:
+    - Implement patch method
+    """
+
+    def __init__(self, service_url: str, is_draft: bool, **kwargs):
+        self._service_url = service_url
+        self._is_draft = is_draft
+
+    def _send_request(self, url: str, method: str, data: Dict, headers: Dict = {}):
```

Comment:
weird to me that you have to pass unused data for the GET implementation here. I'd make data = None by default and it'd have to be passed in for POST, but not GET